src/Testing/LeanCode.IntegrationTestHelpers/LeanCodeTestFactory.cs

```diff
@@ -17,9 +18,15 @@ public abstract class LeanCodeTestFactory<TStartup> : WebApplicationFactory<TSta
 {
     protected abstract ConfigurationOverrides Configuration { get; }
 
-    public virtual JsonSerializerOptions JsonOptions { get; } =
-        new()
+    public virtual JsonSerializerOptions JsonOptions { get; }
+
+    protected virtual string ApiBaseAddress => "api/";
+
+    protected LeanCodeTestFactory()
+    {
+        var options = new JsonSerializerOptions
```

Comment:
You can just assign to `JsonOptions` immediately then do `JsonOptions.MakeReadOnly();`.